cypress/page/products-list.page.ts

```diff
@@ -0,0 +1,25 @@
+class ProductsListPage {
+    private productNumber: number
+    private addToCartButton: string
+    private addToCartButtonContainer: string
+    private addToCartButtonText: string
+
+    constructor(productNumber: number) {
```

Comment:
This shouldn't be in the constructor, the parameter should go on the addToCart function